packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -387,9 +387,14 @@ class ListTile extends StatelessWidget {
     // Overall, the list tile is a Row() with these children.
     final List<Widget> children = <Widget>[];
 
+    IconThemeData iconThemeData;
+    if (leading != null || trailing != null) {
+      iconThemeData = new IconThemeData(color: _iconColor(theme, tileTheme));
+    }
```

Comment:
nit: our [style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo) says to avoid the braces for one-line if statements.

Reply:
looks like you missed this one